Source/missiles.cpp

```diff
@@ -3609,8 +3598,7 @@ void MI_Firemove(MissileStruct &missile)
 {
 	constexpr int ExpLight[14] = { 2, 3, 4, 5, 5, 6, 7, 8, 9, 10, 11, 12, 12 };
 
-	missile.position.tile.x--;
-	missile.position.tile.y--;
+	missile.position.tile -= Displacement { 1, 1 };

```

Comment:
```suggestion
```
`missile.position.tile` gets set in UpdateMissilePos, so this modification does nothing before the next use. Same with the modification to `missile.position.offset.deltaY`.